src/ProductConstructionService/Readme.md

```diff
@@ -0,0 +1,17 @@
+# Starting the service locally
+
+To run the Product Construction Service locally, set the `ProductConstructionService.AppHost` as Startup Project, and run with F5
+
+# Instructions for recreating the Product Construction Service
+Using Azure CLI, run the following lines:
+ - `az account set --subscription <subscription name/id>`
+ - `az group create --location northcentralus --name product-construction-service`
```

Comment:
Should it be this?
```suggestion
 - `az group create --location northcentralusstage --name product-construction-service`
```

Reply:
Also please put a comment why this location

Reply:
I think that step might not be doable with azure cli at all, as it has input validation that does not include that location.
Also creating a RG on the portal doesn't allow you to create a resource group with that location. Do we need a special feature flag in our subscription?

Reply:
I sent a question to the manager there asking if we can use this in other locations already

Reply:
My experimental app has this YAML description:
```yaml
id: /subscriptions/0b38483d-f273-4816-832b-59fea3a17a23/resourceGroups/backflow-service/providers/Microsoft.App/containerapps/backflow-service-app
identity:
  type: None
location: North Central US (Stage)
name: backflow-service-app
```

Reply:
we probably won't be able to create the resource group with azure CLI, I'll update the instructions. Hopefully bicep doesn't give us trouble with that.

Reply:
I am not sure what is the right string for that location, yeah

Reply:
I modified the provisioning script as we talked about. We now create the Container Environment and the app in the North Central US Stage location, while the rest of the resources is in the normal, North Central US location